cob_hardware_config/raw3-5/config/base_controller.yaml

```diff
@@ -20,17 +21,21 @@ twist_controller:
       spring: 15.0 #10.0
       damp: 2.75 #2.5
       virt_mass: 0.1 #0.1
-      d_phi_max: 12.0 #100.0
+      d_phi_max: 10.0 #100.0
       dd_phi_max: 40.0 #100.0
   wheels: &wheels
     - steer: fl_caster_rotation_joint
       drive: fl_caster_r_wheel_joint
+      steer_neutral_position: -180.0
```

Comment:
note that you can also define what the elmo considers to be positive/negative turning direction in the controller itself. @ipa-mdl is this configured somewhere in those files? If not, this could be the issue here that this has been flashed to the elmo.